src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const list = document.querySelector('.gallery__list');
+const bigImage = document.querySelector('#largeImg');
+
+list.addEventListener('click', (evt) => {
```

Comment:
```suggestion
list.addEventListener('click', (event) => {
```
it's always better to avoid short forms - FYI